python_modules/libraries/dagster-dbt/dagster_dbt_tests/conftest.py

```diff
@@ -17,8 +17,10 @@
     test_dbt_unit_tests_path,
     test_duplicate_source_asset_key_path,
     test_jaffle_shop_path,
+    test_last_update_freshness_path,
```

Comment:
To make this easier to review, could you add the two base jaffle shop projects in a separate PR, similar to https://github.com/dagster-io/dagster/pull/22246?

It's hard to tell what has changed in the dbt projects that you're testing.